waymore: add page (#12487)
tldr-pages/tldr: pages/common/waymore.md

# waymore

> Fetch URLs of a domain from Wayback Machine, Common Crawl, Alien Vault OTX, URLScan, and VirusTotal.
> Note: Unless specified, output is dumped into `results/` directory where waymore `config.yml` resides (by default in `~/.config/waymore/`).
> More information: <https://github.com/xnl-h4ck3r/waymore>.

- Search for URLs of a domain (output will typically be in `~/.config/waymore/results/`):

`waymore -i {{example.com}}`

> - Limit search results to only include list of URLs for a domain, with outputs sent to a specific file:

Reviewer: ```suggestion
- Limit search results to only include a list of URLs for a domain and store outputs to the specified file:
```

`waymore -mode U -oU {{path/to/example.com-urls.txt}} -i {{example.com}}`

> - Limit search results to only include content body of URLs of a domain, with outputs sent to a specific directory:

Reviewer: ```suggestion
- Only output the content bodies of URLs and store outputs to the specified directory:
```

`waymore -mode R -oR {{path/to/example.com-url-responses}} -i {{example.com}}`

> - Limit search to limit results within specific dates:

Reviewer: ```suggestion
- Filter the results by timestamp:
```

Author: The word _timestamp_ here does not sound right to me; the switches used are `-to` and `-from`. How about:

```
Filter the results by specifying date ranges:
```
